### ISSUE TITLE
Bridge for `UnsafeMutablePointer<CGPoint>`

### ISSUE THREAD

**Issue by gapl** [ReactiveKit/Bond#481]
I am getting the following `fatal error` when trying to convert a protocol method into signal:
```
fatal error: Bridging ObjC type `BNDNSObjCValueType(rawValue: 123)` is not supported.: file .../Bond/Sources/Bond/ProtocolProxy.swift, line 75
```

This stopped working in Bond v6.4.3. It was working in a previous version of Bond I was using from commit hash "fffa4f6995c2cf594b29752ee3405c7a1806cb1f".

I'm converting the following protocol method in such a way:
```
var reachedBottom: SafeSignal<Void> {
    let dispatch = { (subject: SafePublishSubject<Void>, scrollView: UIScrollView, _: CGPoint, targetContentOffset: UnsafeMutablePointer<CGPoint>) in
        let maxOffset = scrollView.contentSize.height - scrollView.bounds.height
        let targetOffset = targetContentOffset.pointee.y
        if targetOffset >= maxOffset {
            subject.next(())
        }
    }
    return reactive.delegate.signal(for: #selector(UIScrollViewDelegate.scrollViewWillEndDragging(_:withVelocity:targetContentOffset:)), dispatch: dispatch)
}
```

I'm assuming the error comes from missing bridge for `UnsafeMutablePointer<CGPoint>`.

**Comment by debbiegoldsmith** [comment]
The new, type-safe extension of BNDInvocation in ProtocolProxy.swift does not handle structure types (NSObjCStructType). Unfortunately, quite a few delegate selectors in AppKit and UIKit use structure types like NSPoint and CGPoint. Those used to work in the old version of the proxy, but now they get a fatal runtime error with "unsupported type". Is there any possibility of re-supporting structure types in protocol proxies?

**Comment by codedbypm** [comment]
I think I hit the same issue when handling NSRange types in UITextViewDelegate's method `textView(_:shouldInteractWith:in:interaction:)`. Any clue?

**Comment by srdanrasic** [comment]
Thanks everyone for reporting this. I've just released 6.4.4 that should fix remaining types.
